docs/section21/Rule21-3.md

```diff
@@ -0,0 +1,53 @@
+
+# Boiler - Rule 21-3  
+
+**Rule ID:** 21-3  
+**Rule Description:** Heating hot water plant capacity shall be based on coincident loads.  
+**Rule Assertion:** B-RMR FluidLoop.heating_design_and_control: is_sized_using_coincident_load = True  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.2.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B-RMR  
+**Applicability Checks:**  
+
+1. Pass Rule 21-1.
+2. B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11, 12 or that is Type-1, 5, 7, 11, 12.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:**  
+
+1. check_purchased_chw_hhw()
+
+**Applicability Check:**
+
+1. Pass Rule 21-1: `if rule-21-1 == "PASS":`
+
```

Comment:
This should be calling ceck_purchase_chw_hw function instead of referring to the rule.

Reply:
Updated.